helm/reana/README.md

```diff
@@ -98,6 +98,12 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `reana_hostname`                                         | REANA hostname (e.g. reana.example.org)                                              | None                                            |
 | `namespace_runtime`                                      | Namespace in which the REANA runtime pods (workflow engines, jobs etc...) will run   | `.Release.Namespace`                            |
 | `naming_scheme`                                          | REANA component naming scheme                                                        | None                                            |
+| `pgbouncer.enabled`                                      | Instantiate PgBouncer inside the cluster to pool database connections                | false                                           |
```

Comment:
When `pgbouncer` is enabled, shall we act on `reana-job-controller`'s `
REANA_DB_CLOSE_POOL_CONNECTIONS` configuration value?

Reply:
No, it's actually the opposite, if pgbouncer is not enabled then the two options are:
- increase the number of connection slots of the database
- close the connections as soon as they are returned to SQLAlchemy's connection pool
I have kept the default value of `REANA_DB_CLOSE_POOL_CONNECTIONS` as to have the best performance possible, but it is customisable so that we can revert back the change in case of troubles with PgBouncer. 

What do you think we should do regarding this? Shall we add this to the documentation?

Reply:
Yes, we can later improve the scalability-related documentation regarding PgBouncer.